stages/org.osbuild.systemd.unit

```diff
@@ -60,12 +77,18 @@ def main(tree, options):
     unit = options["unit"]
     dropin_file = options["dropin"]
     cfg = options["config"]
+    systemd_options = options["systemd-options"]
 
     # ensure the unit name + ".d" does not exceed maximum filename length
     if len(unit+".d") > 255:
         raise ValueError(f"Error: the {unit} unit drop-in directory exceeds the maximum filename length.")
 
-    unit_dropins_dir = f"{tree}/usr/lib/systemd/system/{unit}.d"
+    match systemd_options:
```

Comment:
This requires python 3.10, do we need to consider the backward compatibility?  @achilleas-k @thozza

Reply:
Yes, osbuild still needs to run on RHEL 8, and I think RHEL 8.9 is going to have 3.6.